### DISCUSSION TITLE
Camera feed seems to be inaccessible

### DISCUSSION THREAD

**Discussion by guruprasadah** [XiaoMi/ha_xiaomi_home#78] [Q&A] [answered]
Im using a Mi Home Security Camera 360, and all the entities show up, but there seems to be no video feed. is this intentional, will there ever be an accessible video feed, or never?

**Comment by SusanPhevos** [comment, marked as answer]
xiaomi_home does not support getting videos or pictures from the camera by design. This function will never be implemented.

**Reply by yongli-abc** [reply]
What? At least provide some rationales please?

**Reply by williamchenghu** [reply]
according to [another answer](https://github.com/XiaoMi/ha_xiaomi_home/discussions/232#discussioncomment-11668543) in #232 :
_it's told that the stream is encrypted e2e, can only be decrypted at the Mi Home app. hence, the feature asked above (enable picture / video stream to end users) wasn't able to be realized back then, not now, not in the coming future either._

p.s. this answer is eye-opening for real.
> xiaomi_home does not support getting videos or pictures from the camera by design. This function will never be implemented.

reasoning coming from official maintainer team might not always be a cheerful one, we understand that. We do however expect sth bouncing back than merely hearing "it'll never be implemented" and call it a day. As a maintainer, your ways of "maintaining" the community is absolutely stunning. Your attitude and tone of voice is jaw-droppingly amazing, unfortunately in the wrong direction. 😔

**Reply by Rhaegarlai** [reply]
正当小米计划推出NAS产品的同时，又在给人找不要买NAS的理由，云服务是好，但是云服务并不是永远工作的。这里说的不仅仅是小米的云服务器，家里的网络有时也会故障的。

**Comment by e7naq3y** [comment]
就你妈逼想钱想疯了,逼着用户用您那傻逼的云服务,草泥马的,早晚给你全破解刷机

**Reply by Warm-rain** [reply]
另外一个可以啊

**Comment by Michaeljczhang** [comment]
Streaming is supported by go2rtc now.
https://github.com/AlexxIT/go2rtc
